src/Service/PopulateService.php

```diff
@@ -58,82 +62,92 @@ public function __construct(EntityManagerInterface $entityManager, SearchReposit
      *
      * @return void
      */
-    public function populate(string $index, int $record_id = -1): void
+    public function populate(int $record_id = -1): void
```

Comment:
If we plan to run this at intervals we should do a lock similar to vendor imports to ensure we don't run multiple populates in parallel.

Reply:
Lock added

Reply:
But in the Command, not the Service. What if the Service is use some place else...